.github/workflows/nudgebot.yaml

```diff
@@ -0,0 +1,37 @@
+name: Label inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  label-inactive-issues:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'pantsbuild'
```

Comment:
This makes testing in forks difficult, but, like you said, this is dealing with issues in the main repo, so testing in a fork might not provide much value.